sessions/forecasting-models.qmd

```diff
@@ -31,15 +38,14 @@ The source file of this session is located at `sessions/forecasting-models.qmd`.
 
 ## Libraries used
 
-In this session we will use the `nfidd` package to load a data set of infection times and access stan models and helper functions, the `dplyr` and `tidyr` packages for data wrangling, `ggplot2` library for plotting, the `tidybayes` package for extracting results of the inference and the `scoringutils` package for evaluating forecasts.
+In this session we will use the `fpp3` package to fit and forecast ARIMA models, the `dplyr` and `tidyr` packages for data wrangling, and `ggplot2` library for plotting.
```

Comment:
aren't we really using fable

Reply:
yes, but there are other fiddly bits that fpp3 loads that I was being lazy about loading individually. but I see the point. we should probably load them individually.